.travis.yml

```diff
@@ -9,6 +9,13 @@ python:
   - "pypy"
   - "pypy3"
 
+# work-around for Python 3.7 which is on a non-standard Travis Ubuntu release (Xenial)
+matrix:
+  include:
+    - python: 3.7
+      dist: xenial
+      sudo: true
```

Comment:
Why does this need sudo?

Reply:
It's due to this Travis oddity: https://github.com/travis-ci/travis-ci/issues/10110 .